plugins/destiny.py

```diff
@@ -23,6 +24,8 @@
     'secondsPlayed', 'longestSingleLife', 'orbsDropped', 'precisionKills',
     'precisionRate', 'suicides', 'winRate', 'publicEventsCompleted']
 
+# Reset is at 9am UTC, so -9h UTC makes the reset midnight
+DAILY_RESET_TIMEZONE = datetime.timezone(-datetime.timedelta(hours=9))
```

Comment:
It would be better to remove this in favor of a better approach (below). Also, datetime has no method timezone. You may be thinking of the time module.